packages/editor/CodeMirror/createEditor.ts

```diff
@@ -268,8 +268,6 @@ const createEditor = (
 					},
 				}),
 
-				EditorState.tabSize.of(4),
-
```

Comment:
Related notes:
- `tabSize` should [default to 4](https://github.com/codemirror/state/blob/64cba4b48555891636bc9afba74952c4224f53b1/src/state.ts#L268), so this configuration line was probably not previously necessary.
- `tabSize` does seem to be used for internal CodeMirror calculations [in at least one place (I think for guessing indentation depth with mixed tabs and spaces)](https://github.com/codemirror/commands/blob/d6b6d01c470ab6e82c01d084474c7007a5e7c64e/src/commands.ts#L805) (see [search results for tabSize in org:codemirror](https://github.com/search?q=org%3Acodemirror+tabSize&type=code&p=4)).